space-cli/cmd/modules/accounts/view.go

```diff
@@ -0,0 +1,49 @@
+package accounts
+
+import (
+	"os"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/spaceuptech/space-cli/cmd/model"
+	"github.com/spaceuptech/space-cli/cmd/utils"
+)
+
+func listAccounts(accountID string, showKeys bool) error {
+	credential, err := utils.GetCredentials()
+	if err != nil {
+		return err
+	}
+
+	var accounts []*model.Account
+	if accountID != "" {
+		for _, v := range credential.Accounts {
+			if v.ID == accountID {
+				accounts = append(accounts, v)
```

Comment:
break the loop when you found accountid, 
@YourTechBud do you think that we should log some message when we couldn't find provided account id, showing all